app/lib/db/migrations/0005_watery_runaways.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE `access_tokens` (
+	`id` text PRIMARY KEY NOT NULL,
+	`user_id` text NOT NULL,
+	`token` text NOT NULL,
+	`created_at` integer NOT NULL,
+	`updated_at` integer NOT NULL
+);
+--> statement-breakpoint
+CREATE TABLE `support_tickets` (
```

Comment:
I'm not fully sure why this creeeps in.  I was just using autogen.  My guess is cause there's an existing 0006 migration already?

Reply:
Ya, looking at the journal below the 0006 migration isn't even picked up.